packages/gitbook/src/components/PageAside/ScrollSectionsList.tsx

```diff
@@ -7,6 +7,7 @@ import type { DocumentSection } from '@/lib/document-sections';
 import { tcls } from '@/lib/tailwind';
 
 import { HEADER_HEIGHT_DESKTOP } from '../layout';
+import { useBodyLoaded } from '../primitives/LoadingStateProvider';
```

Comment:
I think we should do more import as:

```suggestion
import { useBodyLoaded } from '@/components/primitives';
```